lhm.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.3.0'
 
-  s.add_dependency 'retriable', '>= 3.0.0'
+  s.add_dependency 'retriable'
```

Comment:
This seems a _tiny_ bit dangerous, but I suspect if someone is running a ridiculously old `retriable` they will come to us with errors anyway.